test/modules/standard/IO/saru/getParentName/parentName.chpl

```diff
@@ -0,0 +1,10 @@
+use IO;
+
+/* Tests IO.file.getParentName() function */
+proc main() {
+  var f = open('parentNameTest.txt', iomode.r),
+      c = f.reader(),
+      err: syserr;
+
+  writeln(f.getParentName(err));
```

Comment:
I believe you can't open a file object over a directory at the moment.  What about a file opened using a relative path?